examples/s2-parcel-example/src/App.js

```diff
@@ -35,9 +35,14 @@ import {
   TableView,
   Text,
   ToggleButton,
-  ToggleButtonGroup
+  ToggleButtonGroup,
+  TreeView,
+  TreeViewItem,
+  TreeViewItemContent
 } from "@react-spectrum/s2";
 import Edit from "@react-spectrum/s2/icons/Edit";
+import FileTxt from "@react-spectrum/s2/icons/FileTxt";
```

Comment:
should the import be "@react-spectrum/s2/icons/FileText"? i ran into an issue when trying to run the app cause it couldn't find the import and when i looked through the icons folder, it looked like it was called Text instead of Txt